src/plugins/api/index.ts

```
@@ -1454,33 +1454,47 @@ export class MusicAssistantApi {
   }
 
   private handleResultMessage(msg: SuccessResultMessage | ErrorResultMessage) {
-    // Handle result of a command
     const resultPromise = this.commands.get(msg.message_id as number);
 
+    const isAuthAborted =
+      "error_code" in msg &&
+      typeof msg.details === "string" &&
+      msg.details === "auth_aborted";
+
+    const isAuthTimeout =
+      "error_code" in msg &&
+      typeof msg.details === "string" &&
+      msg.details === "auth_timeout";
+
     if ("error_code" in msg) {
-      // always handle error (as we may be missing a resolve promise for this command)
       msg = msg as ErrorResultMessage;
-      console.error("[resultMessage]", msg);
-      store.activeAlert = {
-        type: AlertType.ERROR,
-        message: msg.details || msg.error_code,
-        persistent: false,
-      };
-    } else if (DEBUG) {
-      console.log("[resultMessage]", msg);
```

Suggestion to make the error bubble up towards the consumer is an extra parameter to set that we want the exception to bubble up and then keep the fallback catch all exception handler with the snackbar?